packages/rule-utils/src/colors.ts

```diff
@@ -58,9 +55,11 @@ export function colorToString(color: CSSColorValue | string, alphaOverride?: str
   alpha = alphaOverride ?? alpha
   type = type.toLowerCase()
 
-  // Comma separated functions
-  if (['hsla', 'hsl', 'rgba', 'rgb'].includes(type))
-    return `${type.replace('a', '')}a(${components.join(',')}${alpha == null ? '' : `,${alpha}`})`
+  if (['hsla', 'rgba'].includes(type))
+    return `${type}(${components.join(', ')}${alpha == null ? '' : `, ${alpha}`})`
+
+  if (['rgb', 'hsl'].includes(type))
```

Comment:
This condition and its branch is no longer necessary. The continuation after will format the rgb and hsl as usual

Reply:
In tailwind play, it will generate to `rgb`, if remove condition that `bg-red-500` will be generated to `rgba`
<img width="545" alt="image" src="https://github.com/unocss/unocss/assets/57086651/533233a1-ca86-41c7-a21d-ecc175f0c25c">